source/likely.js

```diff
@@ -4,6 +4,7 @@ var likely = require('./index.js');
 var history = require('./history');
 
 window.addEventListener('load', function () {
+    likely.defaults = likely.getDefaultOptions();
```

Comment:
As far as I see, `likely.defaults` is used in only one place – [`index.js:23`](https://github.com/ilyabirman/Likely/blob/master/source/index.js#L23). Is setting it necessary?